fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -81,19 +81,45 @@ def self.list_available_developer_id_installer(in_keychain: nil)
       `#{commands.join(' ')}`
     end
 
-    def self.wwdr_certificate_installed?
-      certificate_name = "Apple Worldwide Developer Relations Certification Authority"
-      certificate_hash = "SHA-256 hash: BDD4ED6E74691F0C2BFD01BE0296197AF1379E0418E2D300EFA9C3BEF642CA30"
+    def self.installed_wwdr_certificates
+      certificate_name = "Apple Worldwide Developer Relations"
+
+      # Find all installed WWDRCA certificates
+      installed_certs = []
+      Helper.backticks("security find-certificate -a -c '#{certificate_name}' -p #{wwdr_keychain.shellescape}")
+            .lines
+            .each do |line|
+        if line.start_with?('-----BEGIN CERTIFICATE-----')
+          installed_certs << line
+        else
+          installed_certs.last << line
+        end
+      end
 
-      keychain = wwdr_keychain
-      response = Helper.backticks("security find-certificate -a -c '#{certificate_name}' -Z #{keychain.shellescape} | grep ^SHA-256", print: FastlaneCore::Globals.verbose?)
+      # Get 'organisational unit' of the installed WWDRCA certificates
+      installed_certs.map do |pem|
+        OpenSSL::X509::Certificate.new(pem).subject.to_a.find { |part| part[0] == 'OU' }[1]
```

Comment:
I love how Apple inconsistently uses which field they put what information in 😅 I wonder if it would make more sense to look at the SHA of the certificates though, since that should always be consistent... so even if Apple releases a G7 tomorrow that uses `Apple Certification Authority` as the `OU`, this code would continue to work as expected.

Reply:
Addressed in https://github.com/fastlane/fastlane/pull/20537/commits/6c5d2b916956f20ea76aadcfdffc70c2ed0464f4